commit 4fef3ef182f03bbfe498d1f0605667a33a4b2849
Author: Daniel Kolesa <dkolesa@igalia.com>
Date:   Tue Apr 12 15:45:35 2022 +0200

    disable exception handler test
    
    This one has strange behavior and does not actually fail as
    is the expectation.

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index 8d53f51..ef3f967 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -1,6 +1,6 @@
 project(tests C)
 
-set(TEST_EXCEPTION_HANDLER TRUE)
+set(TEST_EXCEPTION_HANDLER FALSE)
 if (CMAKE_BUILD_TYPE)
     string(TOLOWER "${CMAKE_BUILD_TYPE}" CMAKE_BUILD_TYPE_LOWER)
     if (CMAKE_BUILD_TYPE_LOWER STREQUAL "undefinedsanitizer")
